jmmorrone/master-recipe
# Added a simple cache for search and a unit test

## What did you implement:
* A simple cache layer to improve search.
* A unit test for demo.

## How did you implement it:
* Used a simple memory cache layer over search endpoint.

## How can we verify it:
* If you have this on your local, check ms in the logs when performing a search action.
